types/bellatrix/execution_payload.yaml

```diff
@@ -0,0 +1,49 @@
+Bellatrix:
+  ExecutionPayloadCommon:
+    # An abstract object to collect the common fields between the ExecutionPayload and the ExecutionPayloadHeader objects.
+    type: object
+    description: "The [`ExecutionPayload`](https://github.com/ethereum/eth2.0-specs/blob/v1.1.9/specs/bellatrix) object from the Eth2.0 Bellatrix spec."
+    properties:
+      parent_hash:
+        $ref: '../primitive.yaml#/Root'
+      fee_recipient:
+        $ref: '../primitive.yaml#/ExecutionAddress'
+      state_root:
+        $ref: '../primitive.yaml#/Root'
+      receipts_root:
+        $ref: '../primitive.yaml#/Root'
+      logs_bloom:
+        $ref: '../primitive.yaml#/LogsBloom'
+      prev_randao:
+        $ref: '../primitive.yaml#/Root'
```

Comment:
Is this item a root?  Thought it was the mix, although not 100% sure on that.

Reply:
Yes this is a mix, not a root. We actually use `Root` for `randao_mixes` in `BeaconState` already which I think is why I used it here. But in [this commit](https://github.com/ethereum/beacon-APIs/pull/194/commits/0e93e46248841fd90b266b59607205ac612ef79a) I updated both places to use a new `Bytes32` type. Could also rename `Bytes32` to `RandaoMix` ?